babel.config.js

```diff
@@ -25,55 +25,6 @@ const defaultPlugins = [
   '@babel/plugin-proposal-optional-chaining',
   '@babel/plugin-syntax-dynamic-import',
   'babel-plugin-inline-json-import',
-  [
-    'module-resolver',
-    isESM
-      ? {
-          root: ['./src'],
-          alias: {
-            coreSrc: './src',
-            Actions: './src/actions',
-            App: './src/components/App',
-            Collection: './src/components/Collection',
-            Constants: './src/constants',
-            Editor: './src/components/Editor',
-            EditorWidgets: './src/components/EditorWidgets',
-            Formats: './src/formats',
-            Integrations: './src/integrations',
-            Lib: './src/lib',
-            MediaLibrary: './src/components/MediaLibrary',
-            Reducers: './src/reducers',
-            Selectors: './src/selectors',
-            ReduxStore: './src/redux',
-            Routing: './src/routing',
-            UI: './src/components/UI',
-            Workflow: './src/components/Workflow',
-            ValueObjects: './src/valueObjects',
-            localforage: 'localforage',
-            redux: 'redux',
-          },
-          extensions: ['.js', '.jsx', '.es', '.es6', '.mjs', '.ts', '.tsx'],
-        }
-      : {
-          root: path.join(__dirname, 'packages/netlify-cms-core/src/components'),
-          alias: {
-            coreSrc: path.join(__dirname, 'packages/netlify-cms-core/src'),
-            Actions: path.join(__dirname, 'packages/netlify-cms-core/src/actions/'),
-            Constants: path.join(__dirname, 'packages/netlify-cms-core/src/constants/'),
-            Formats: path.join(__dirname, 'packages/netlify-cms-core/src/formats/'),
-            Integrations: path.join(__dirname, 'packages/netlify-cms-core/src/integrations/'),
-            Lib: path.join(__dirname, 'packages/netlify-cms-core/src/lib/'),
-            Reducers: path.join(__dirname, 'packages/netlify-cms-core/src/reducers/'),
-            Selectors: path.join(__dirname, 'packages/netlify-cms-core/src/selectors/'),
-            ReduxStore: path.join(__dirname, 'packages/netlify-cms-core/src/redux/'),
-            Routing: path.join(__dirname, 'packages/netlify-cms-core/src/routing/'),
-            ValueObjects: path.join(__dirname, 'packages/netlify-cms-core/src/valueObjects/'),
-            localforage: 'localforage',
-            redux: 'redux',
```

Comment:
Have no idea, why localforage and redux were aliased. But seems like it works fine without it.